pkg/datastore/db_user.go

```diff
@@ -134,8 +139,8 @@ func getDbUsers(tableName string, withTenantIdCheck, withInstanceIdCheck bool) [
 		dbUsers[i].password = getPassword(string(dbUsers[i].username))
 		dbUsers[i].policyName = getRlsPolicyName(string(dbUsers[i].username), tableName)
 	}
-	TRACE("Returning DB user specs for table %q:\n\twithTenantIdCheck - %t\n\twithInstanceIdCheck -  %t\n\tdbUsers - %+v\n",
-		tableName, withTenantIdCheck, withInstanceIdCheck, dbUsers)
+	TRACE("Returning DB user specs for table %q:\n\t[roleT=%s, roleI=%s, tableT=%s, tableI=%s]\n\tdbUsers - %+v\n",
+		tableName, tenantRole, instancerRole, isTableTenanted, isTableInstanced, dbUsers)
```

Comment:
`tenantRole`, `instancerRole`, `isTableTenanted`, `isTableInstanced` are all boolean variables. They need to be printed using `%t` placeholder, not `%s`.
Otherwise, something like that is printed: `roleI=%!s(bool=false)`

Reply:
noted will look into it later